ui/src/app/views/workspace/component.ts

```diff
@@ -32,7 +35,22 @@ export class WorkspaceComponent implements OnInit {
     this.cohortsService.getCohortsInWorkspace('123', '123').subscribe(
           cohortsReceived => {
             this.cohortList = cohortsReceived;
+            this.CohortEditService.list().then(
+              cohorts => {
+                for (const coho of cohorts) {
+                  this.cohortList.push(coho);
```

Comment:
Yes, the end goal is that this is just special logic for the demo. 
In the long run, I don't imagine we will want to store any cohort data in the UI. Editing or adding a cohort will post that data to the server, and loading them in they will all come through the API from the server. 
I just wanted to create the basics of what the UI will look like, and display the sort of workflow a user would go through if they wanted to add or edit a cohort.